app/auth/mutations/signup.ts

```diff
@@ -85,11 +85,11 @@ export default resolver.pipe(
       }),
       ctx.session.$create({
         userId: user.id,
-        roles: [user.role, user.memberships[0]!.role],
+        role: user.role,
         workspaceId: user.memberships[0]!.workspaceId,
       }),
     ])
 
     return user
-  }
+  },
```

Comment:
It'd be nice if we add to the VS Code config file as to whether we use trailing commans or not 
e.g., `"trailingComma": "none"`

Reply:
Thanks - it seems like this snuck in here. Happy to discuss and track in an issue #1989.